boards/limifrog-v1/dist/openocd.cfg

```diff
@@ -0,0 +1,5 @@
+source [find interface/stlink-v2.cfg]
+transport select hla_swd
+
+set WORKAREASIZE 0x4000
```

Comment:
In the regarding file I found:
```
# Work-area is a space in RAM used for flash programming
# By default use 10kB
if { [info exists WORKAREASIZE] } {
   set _WORKAREASIZE $WORKAREASIZE
} else {
   set _WORKAREASIZE 0x2800
}
```
So forget about the 80k above and let's initially go with `0x2800` :-)